packages/react-day-picker/src/contexts/Focus/utils/getNextFocus.tsx

```diff
@@ -59,7 +73,22 @@ export function getNextFocus(
     const isFocusable = !activeModifiers.disabled && !activeModifiers.hidden;
 
     if (!isFocusable) {
-      return getNextFocus(newFocusedDay, moveBy, direction, options, modifiers);
+      if (retryOptions.attempt >= retryOptions.attemptLimit) {
+        const origDayString = format(retryOptions.originalDay, 'P');
+        const warning = `Unable to select day within ${retryOptions.attemptLimit} days of ${origDayString}`;
+        // eslint-disable-next-line no-console
+        console.warn(warning);
```

Comment:
I'm not sure if this is a pattern you'd like to adopt @gpbl, but I figured _something_ should happen, since this just returns the last valid day: `retryOptions.originalDay`

Reply:
I was waffling between `console.debug` and `console.warn` but ended up with warn because it's more likely to be seen and could very well help people debug oddities in the future.